src/main/java/com/puppycrawl/tools/checkstyle/JavadocDetailNodeParser.java

```diff
@@ -163,7 +233,16 @@ private ParseTree parseJavadocAsParseTree(String blockComment) {
         // By default it uses Error Recover Strategy which is slow and useless.
         parser.setErrorHandler(new JavadocParserErrorStrategy());
 
-        return parser.javadoc();
+        final ParseTree javadoc = parser.javadoc();
+        if (errorListenerOption == ErrorListenerOption.NOTIFY_FOR_NON_TIGHT_HTML) {
+            final ParserRuleContext nonTightTagStartContext = parser.nonTightTagStartContext;
+            if (nonTightTagStartContext != null) {
+                final Token offendingToken = ((TerminalNode) nonTightTagStartContext.getChild(1))
+                        .getSymbol();
+                parser.notifyErrorListeners(offendingToken, MSG_UNCLOSED_HTML, null);
```

Comment:
Updated this. Previously I was directly modifying `errorlistener.errorMessage`.

@Vladlis Please have a look. Thank you.